whereby/sdk
# common: Add Provider and set speaker action

### Description

**This is a breaking change**
Adds a provider so that we can access the store in all components and hooks. This changes the API, and adds a requirement that the react hooks are rendered inside of a provider. That's why this is going to the development branch, and we will wait with releasing this until we are ready for `v3.0.0`. 

**Summary:**



**Related Issue:**



### Testing
1. Start storybook: `yarn build && yarn dev`
2. Go to one of the local media stories
3. Verify that the speaker selection works



### Screenshots/GIFs (if applicable)



### Checklist

        under the terms of the MIT license.

### Additional Information
